libs/api/database/src/lib/content/stores/blogs.store.ts

```diff
@@ -75,45 +74,14 @@ export class BlogsStore {
      * @param blogId The blog's ID
      * @param pubChange Object for change in publishing status
      */
-    async changePublishStatus(user: JwtPayload, blogId: string, pubChange: PubChange): Promise<BlogsContentDocument> {
-        if (pubChange.oldStatus === PubStatus.Unpublished && pubChange.newStatus === PubStatus.Published) {
-            await this.usersService.changeBlogCount(user, true);
-        } else if (pubChange.oldStatus === PubStatus.Published && pubChange.newStatus === PubStatus.Unpublished) {
-            await this.usersService.changeBlogCount(user, false);
-        }
-
+    async changePublishStatus(user: string, blogId: string, pubChange: PubChange): Promise<BlogsContentDocument> {
         return this.blogsModel.findOneAndUpdate(
```

Comment:
There's no longer need to call changeBlogCount()?

Reply:
that function didn't really work as intended even before this change, so we should handle this plus works count in a later PR.